indico/htdocs/sass/partials/_inputs.scss

```diff
@@ -286,12 +286,16 @@ $dropdown-transition: $dropdown-transition-step ease-out;
 }
 
 .i-radio, .i-checkbox {
-    white-space: nowrap;
+    display: flex;
+    input {
```

Comment:
We tend to leave an empty line between CSS rules and selectors, but it's not something important!